### PR TITLE
[DO-2077] Export pep view

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

File: business_register/views/pep_views.py [hunk 1]
```diff
@@ -56,7 +56,34 @@ def retrieve_by_source_id(self, request, pk):
...
+            'Full Name': ['fullname', 30],
+            'Full Name (english transcription)': ['fullname_transcriptions_eng', 36],
+            'Status': ['is_pep', 10],
+            'PEP Type': ['pep_type', 10],
+            'Date of Birth': ['date_of_birth', 19],
+            'Created Date': ['created_at', 19],
+            'Updated Date': ['updated_at', 19],
+            'Last Job Title': ['last_job_title', 20],
+            'Last Employer': ['last_employer', 20]
+        }
```
Comment (roman-tiukh) [inline business_register/views/pep_views.py:76]: needs localization

File: business_register/forms.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ def clean(self):
             return cleaned_data
 
 
-class PepExportForm(PeriodLimitationForm):
+class PepExportUpdatesForm(PeriodLimitationForm):
     selection_field = 'updated_at'
     days_limit = settings.PEP_EXPORT_XLSX_DAYS_LIMIT
```
Comment (roman-tiukh) [inline business_register/forms.py:32]: in this case we don't need PeriodLimitationForm

File: business_register/views/pep_views.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/views/pep_views.py]: done

**Review by roman-tiukh — APPROVED** [review]

File: business_register/models/fop_models.py [hunk 3]
```diff
@@ -21,7 +21,7 @@ class Fop(DataOceanModel):
                                                max_length=275, null=True, blank=True)
     contact_info = models.TextField(_('contacts'), null=True, blank=True)
     vp_dates = models.CharField(max_length=400, null=True, blank=True)
-    authority = models.ForeignKey(Authority, on_delete=models.CASCADE,
+    authority = models.ForeignKey(Authority, related_name='authority', on_delete=models.CASCADE,
```
Comment (IegorSoboliev) [inline business_register/models/fop_models.py:24]: maybe, 'registered_fops'?

Comment (roman-tiukh) [reply business_register/models/fop_models.py:24]:
```suggestion
    authority = models.ForeignKey(Authority, related_name='registered_fops', on_delete=models.CASCADE,
```

File: business_register/views/fop_views.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 from django.conf import settings
 from django.utils.decorators import method_decorator
-from django.utils.translation import gettext_lazy as _
+from django.utils.translation import gettext_lazy as _, gettext as __
```
Comment (roman-tiukh) [inline business_register/views/fop_views.py:3]: wrong imports...

File: business_register/views/pep_views.py [hunk 5]
```diff
@@ -1,7 +1,7 @@
 from django.conf import settings
 from django.shortcuts import get_object_or_404
 from django.utils.decorators import method_decorator
-from django.utils.translation import gettext_lazy as _
+from django.utils.translation import gettext_lazy as _, gettext as __
```
Comment (roman-tiukh) [inline business_register/views/pep_views.py:4]: wrong import...

File: data_ocean/export.py [hunk 6]
```diff
@@ -18,11 +18,11 @@
...
 
     @staticmethod
-    def export(params, export_dict, model_path, filterset_module, user_id):
+    def export(params, export_dict, model_path, filterset_module, serializer_module, user_id):
         app, model = model_path.split('.')
         queryset = apps.get_model(app, model).objects.all()
-        export_filterset = import_string(filterset_module)
-        queryset = export_filterset(params,  queryset).qs
+        queryset = import_string(filterset_module)(params,  queryset).qs
+        serializer = import_string(serializer_module)(queryset, many=True)
```
Comment (roman-tiukh) [inline data_ocean/export.py:25]: wait... why do we import all this?

**Review by roman-tiukh — APPROVED** [review]

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
blocked

File: business_register/views/fop_views.py (hunk 4 above)
Comment (Iva-khar) [reply business_register/views/fop_views.py:3]: rewrited

File: business_register/views/pep_views.py (hunk 5 above)
Comment (Iva-khar) [reply business_register/views/pep_views.py:4]: rewrited

File: data_ocean/export.py (hunk 6 above)
Comment (Iva-khar) [reply data_ocean/export.py:25]: selery not receive objects

File: business_register/models/fop_models.py (hunk 3 above)
Comment (Iva-khar) [reply business_register/models/fop_models.py:24]: just deleted

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**